### ISSUE TITLE
Normalize data

### ISSUE THREAD

**Issue by florianschwanz** [fom-big-data-bike-path-quality/fom-big-data-bike-path-quality-analytics#16]
Normalize data used for training to a range from 0 to 1. See https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html
